packages/client/src/components/NavBar/index.js

```diff
@@ -8,14 +8,10 @@ import MenuItem from '@material-ui/core/MenuItem';
 import Menu from '@material-ui/core/Menu';
 import AccountCircleIcon from '@material-ui/icons/AccountCircle';
 import PropTypes from 'prop-types';
-import Modal from '../Modal';
-import Wizard from '../Wizard';
 import Button from '../Button';
 import styled from 'styled-components';
 import { FONTS, COLORS } from '../../constants';
-
-
-const Wiz_content = ['page1', <Button>oh boi</Button>, 'page3']
+import MobileNav from './MobileNav';
 
 const TextThemes = {
   fontSize: {
```

Comment:
We use multiples of 8 for the max width here, inconsistent with mobile nav's 5-pt spacing